config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  get 'greetings/random_greeting', to: 'greetings#random_greeting'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  root "greetings#random_greeting"
```

Comment:
- [ ] @Outkarm Kindly consider distinguishing the API route and route for the front end, currently they are the same, making it impossible to access the performance of your react app in the rails app.